### DISCUSSION TITLE
When Mirador loads a language other than English, Title/Label still defaults to English, is this intentional?

### DISCUSSION THREAD

**Discussion by damonc015** [ProjectMirador/mirador#3935] [General]
When Mirador loads a language other than English, Title/Label still defaults to English and I am wondering if this an intended behavior or if this is a bug. For reference, the screenshot shows the Arabic title in English and the title label is also not translated when swapping to Arabic. Thanks for any clarification. 
<img width="210" alt="Screenshot 2024-10-02 at 12 46 25 PM" src="https://github.com/user-attachments/assets/a2162b02-122e-4538-ad7f-75eafc4f2e78">

**Comment by jbaiter** [comment]
Possibly related: https://github.com/ProjectMirador/mirador/pull/3482

**Comment by marlo-longley** [comment]
Hello! Sorry for things under Discussions getting lost. This was fixed in https://github.com/ProjectMirador/mirador/pull/4054
I hope you were able to get on a newer version and solve your issue. Thanks for flagging this.
